src/bindgen/bindings.rs

```diff
@@ -228,16 +252,35 @@ impl Bindings {
         }
         if self.config.include_version {
             out.new_line_if_not_start();
-            write!(
-                out,
-                "/* Generated with cbindgen:{} */",
-                crate::bindgen::config::VERSION
-            );
+            match self.config.language {
+                Language::C | Language::Cxx => {
+                    write!(
+                        out,
+                        "/* Generated with cbindgen:{} */",
+                        crate::bindgen::config::VERSION
+                    );
+                }
+                Language::Cython => {
+                    write!(
+                        out,
+                        "''' Generated with cbindgen:{} '''",
+                        crate::bindgen::config::VERSION
+                    );
+                }
+            }
+
             out.new_line();
         }
         if let Some(ref f) = self.config.autogen_warning {
             out.new_line_if_not_start();
-            write!(out, "{}", f);
+            match self.config.language {
```

Comment:
I think we should keep these as it was, this contents are controllable by the user so we shouldn't try to mangle them.